docs/plans/2026-03-22-employee-mobile-experience-plan.md

```
@@ -0,0 +1,933 @@
+- [ ] **Step 3: Commit**
+
+```bash
+git add tests/e2e/employee-mobile.spec.ts
+git commit -m "test: add E2E tests for employee mobile experience"
+```
+
+---
+
+## Task Dependency Graph
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)